docs/admin/src/widgets/StringWithInstructions.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const instructionsContainerStyle = {
+  'color': '#444a57',
+  'padding': '20px 0 10px'
+};
+
+const instructionsLinkStyle = {
+  'color': '#444a57',
+  'font-size': '16px',
+  'font-weight': 'normal',
+  'text-decoration': 'underline'
+};
```

Comment:
Would it make sense to move this to a stylesheet and import the stylesheet here so that the CSS is in CSS and not a JS object?

Reply:
This is such a one-off thing that I was avoiding that BUT we do have a stylesheet just for netlify so that seems like a great place for it. 👍

Reply:
It turns out it's easy to add stylesheets to the preview pane but there's no method of adding one to the editing pane. :/